libs/dh/globalization/ui-watt-translation/src/lib/dh-globalization-ui-watt-translation.module.ts

```diff
@@ -39,6 +40,26 @@ export class DhWattClipboardIntlService extends WattClipboardIntlService {
   }
 }
 
+@Injectable()
+export class DhWattPaginatorIntlService extends WattPaginatorIntlService {
```

Comment:
nitpick: rename to `DhPaginatorIntlService`, also the clipboard service above.

Reply:
Done!